forecast-admin/forecast/opportunities/management/commands/load_opportunities.py

```
@@ -0,0 +1,193 @@
+    #         contract_end=cls.parse_date(row[13]),
+    #         contract_year=cls.int_or_fallback(row[11], 1),
+    #         vendor_name=row[7],
+    #         price_min=row[X].replace("\$","").replace(",","")
+    #         price_max=row[X].replace("\$","").replace(",","")
+    #         labor_category=row[1].strip().replace('\n', ' '),
+    #         education_level=cls.model.get_education_code(row[2]),
+    #         min_years_experience=cls.int_or_fallback(row[3]),
+    #         hourly_rate_year1=price,
+    #         hourly_rate_year2=None,
+    #         hourly_rate_year3=None,
+    #         hourly_rate_year4=None,
+    #         hourly_rate_year5=None,
+    #         current_price=display_price,
+    #         next_year_price=None,
+    #         second_year_price=None,
+    #         contractor_site=row[10],
+    #         schedule=schedule,
+    #         business_size=row[8],
+    #         sin=row[0]
+    #     )
+    #
+    #     contract.full_clean(exclude=['piid'])
+    #
+    #     return contract
+    #
+    # @staticmethod
+    # def int_or_fallback(x, fallback=0):
+    #     try:
+    #         return int(x)
+    #     except ValueError:
+    #         return fallback
+    #
+
+    @staticmethod
+    def parse_date(s):
+        if not s or not re.match(r"\d{2}\/\d{2}\/\d{2,4}", s):
+            return None
+        month, day, year = list(map(int, s.split('/')))
+        return date(year, month, day)
+
+    @staticmethod
+    def parse_dollars(s):
+        try:
+            amt = s.strip().replace("$", "").replace(",", "")
+            return Decimal(amt)
+        except:
+            return None
+
+    @classmethod
+    def insert_office(cls, organization, region, replace=True):
+        try:
+            obj, created = Office.objects.update_or_create(
+                organization=organization,
+                region=region
+            )
+            return obj
+        except ValueError:
```

Instead of trapping the exception here and returning False, I would just let the exception bubble up and deal with it in the calling code if you need to. Hiding exceptions like this can be tricky.